elixir/apps/web/lib/web/live/settings/billing.ex

```diff
@@ -166,32 +178,51 @@ defmodule Web.Settings.Billing do
 
     <.section>
       <:title>
-        Enabled Enterprise Features
+        Support
       </:title>
-      <:help>
-        For further details on enrolling in beta features, reach out to your account manager
-      </:help>
       <:content>
-        <.vertical_table id="features">
-          <.vertical_table_row :for={
-            {key, _value} <- Map.delete(Map.from_struct(@account.features), :limits)
-          }>
-            <:label><.feature_name feature={key} /></:label>
-            <:value>
-              <% value = apply(Domain.Accounts, :"#{key}_enabled?", [@account]) %>
-              <.icon
-                :if={value == true}
-                name="hero-check"
-                class="inline-block w-5 h-5 mr-1 text-green-500"
-              />
-              <.icon
-                :if={value == false}
-                name="hero-x-mark"
-                class="inline-block w-5 h-5 mr-1 text-red-500"
-              />
-            </:value>
-          </.vertical_table_row>
-        </.vertical_table>
+        <div class="ml-4 mb-4 text-neutral-600">
+          <span :if={@account.metadata.stripe.support_type == "email"}>
+            Please send
+            <.link
+              class={link_style()}
+              target="_blank"
+              href={
+                mailto_support(
+                  @account,
+                  @subject,
+                  "Support request: #{@account.name}"
+                )
+              }
+            >
+              an email
+            </.link>
+            and we will get back to you as soon as possible.
+          </span>
+
+          <span :if={@account.metadata.stripe.support_type == "email_and_slack"}>
+            Please send us a message in the shared Slack channel or <.link
```

Comment:
@jamilbk please lmk if you have better ideas for the content here

Reply:
I was going to just suggest entering the Slack channel URL directly, then you can link directly to it here.

I think all plans should have email support option, we'll just prioritize using HubSpot.

So you could just have a metadata field `slack_url` or something